### PR TITLE
Playerslist for MasterServer for example(usernames or steamnames)

### PR DESCRIPTION
**Pull request by avdieking**

### CODE REVIEW

**Comment by phalasz** [comment]
I'm not sure I understand what the purpose of this PR is. @avdieking could you add a description on detailing what these changes are supposed to solve?

File: Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs [hunk 1]
```diff
@@ -274,7 +274,7 @@ private void NetworkerDisconnected(NetWorker sender)
 			MasterServerNetworker = null;
 		}
 
-		public void UpdateMasterServerListing(NetWorker server, string comment = null, string gameType = null, string mode = null)
+		public void UpdateMasterServerListing(NetWorker server, string player = null,bool removeplayer = false, string comment = null, string gameType = null, string mode = null)
```
Comment (phalasz) [inline Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs]: new optional parameters should be added to the end of the signature of this method to not cause breaks in exsiting code.

File: Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs [hunk 2]
```diff
@@ -283,7 +283,11 @@ public void UpdateMasterServerListing(NetWorker server, string comment = null, s
 			if (comment != null) registerData.Add("comment", comment);
 			if (gameType != null) registerData.Add("type", gameType);
 			if (mode != null) registerData.Add("mode", mode);
-			registerData.Add("port", new JSONData(server.Port));
+			if (player != null) registerData.Add("players", player);
+			
+
```
Comment (phalasz) [inline Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs]: Lot of unneeded empty lines here.

**Comment by avdieking** [comment]
It's to get a username list in the server browser for example atm it's only a playercount

**Comment by avdieking** [comment]
It doesn't solve anything its just a added feature

**Comment by phalasz** [comment]
I'm not sure that tracking a user name list is the master server's responsibility. Usually that information is retrieved from each server via a query with any other relevant info to be displayed in the server browser list on the user's local client.

**Comment by avdieking** [comment]
Since there is no build in feature for that its the best solution

**Comment by phalasz** [comment]
I personally would implement something similar to https://developer.valvesoftware.com/wiki/Server_queries#Requests for the actual servers instead of this PR for the master server.

**Comment by avdieking** [comment]
I would love to have something like that but I don't have any ideas to do that

**Comment by phalasz** [comment]
Check https://github.com/Unity-Technologies/FPSSample/blob/master/Assets/Scripts/Networking/SQP.cs for an example on how to do it.

**Comment by phalasz** [comment]
Is this relevant? Can this be closed?

**Comment by phalasz** [comment]
Closing this in favour of PR #266 as an additing to that PR's query protocol could provide a player list if needed like the Valve query requests.

**Pull request closed without merging**